tests/unit/test_rust_backend.py

```
@@ -488,9 +526,10 @@ def test_configuration_edge_cases(self, sample_rust_config):
 def test_module_imports():
     """Test that all required modules can be imported."""
     try:
+        import pathlib
```

Ruff (0.11.9)

`pathlib` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)